specs/Storage.Pickers/FileOpenPicker.md

```diff
@@ -63,6 +81,11 @@ using namespace winrt::Microsoft::Windows::Storage::Pickers;
 
 FileOpenPicker openPicker(AppWindow().Id());
 
+// (Optional) set an initial folder by absolute path. 
+//     Takes precedence over SuggestedStartLocation when both defined.
+//     If not found, falls back to SuggestedStartLocation.
+openPicker.SuggestedStartFolder(L"C:\\MyFiles");
```

Comment:
Maybe `SuggestedStartFolderPath` to emphasize that it's a path, and not an `IStorageFolder`.